utils/read_file.go

```diff
@@ -0,0 +1,16 @@
+package utils
+
+import (
+	"fmt"
+	"os"
+)
+
+func ReadFile(filePath string) ([]byte, error) {
```

Comment:
We don't need this function. As you see, it doesn't add any functionality over `os.ReadFile`, so we can just use that instead.